docs/_includes/common/textbox.html

```diff
@@ -135,6 +135,34 @@ <h3 id="textbox-icon">Textbox with Icon</h3>
     {% endhighlight %}
     <p><strong>NOTE:</strong> The icon is presentational, and therefore hidden from assistive technology using <span class="highlight">aria-hidden</span>. Remember, the purpose of the field must be conveyed using a label.</p>
 
+    <h4 id="textbox-icon">With Actionable Icon</h4>
```

Comment:
Can you update the id to `textbox-icon-actionable` (otherwise our HTML validator will complain about unique ids).